tests/forms_tests/tests/test_forms.py

```diff
@@ -2377,3 +2377,53 @@ def test_baseform_repr_dont_trigger_validation(self):
         self.assertRaises(AttributeError, lambda: p.cleaned_data)
         self.assertFalse(p.is_valid())
         self.assertEqual(p.cleaned_data, {'first_name': 'John', 'last_name': 'Lennon'})
+
+    def test_field_name_with_hidden_input(self):
+        """
+        BaseForm._html_output() uses inconsistent formatting for normal row
+        """
+
+        class SomeForm(Form):
+            hidden1 = CharField(widget=HiddenInput)
+            custom = CharField()
+            hidden2 = CharField(widget=HiddenInput)
+
+            def as_p(self):
+                return self._html_output(
+                    normal_row='<p%(html_class_attr)s>%(field)s %(field_name)s</p>',
+                    error_row='%s',
+                    row_ender='</p>',
+                    help_text_html=' %s',
+                    errors_on_separate_row=True)
```

Comment:
According to the definition, all the five parameters are required by the function.And I think the code here is simple and clear.

I will place ``` ) ``` in a new line.Thanks~